frontend-react/src/components/MessageTracker/MessageDetails.test.tsx

```diff
@@ -173,21 +174,66 @@ describe("RSMessageDetail component", () => {
         expect(mockUseMessageDetails).toHaveBeenCalledWith(TEST_ID);
     });
 
-    test("renders expected content", async () => {
+    test("renders expected content", () => {
         mockUseMessageDetails.mockReturnValueOnce({
             messageDetails: MOCK_MESSAGE_DETAIL as RSMessageDetail,
         });
         render(<MessageDetails />);
 
-        expect(screen.getByText(/Message ID/)).toBeInTheDocument();
+        expect(screen.getByText("Message ID")).toBeInTheDocument();
```

Comment:
Small thing, but I think it'd be a good idea to start using `toBeVisible` rather than `toBeInTheDocument` when we're looking for things that are supposed to exist in the DOM.  The former checks for "pseudo-actual" visibility (certain styles like `display`, `visibility`, and opacity) whereas the latter just checks that it exists in the markup.